src/discussions/comments/comment-icons/CommentIcons.jsx

```diff
@@ -0,0 +1,32 @@
+import { faFlag, faStar as faEmptyStar } from '@fortawesome/free-regular-svg-icons';
+import { faEllipsisV, faStar as faSolidStar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function CommentIcons({ abuseFlagged, following }) {
+  return (
+    <div className="d-flex flex-column icons">
+      {/* Only show the star if the comment has a following attribute, indicating it can be followed */}
+      { following !== undefined && (
+        following
+          ? <FontAwesomeIcon icon={faSolidStar} />
+          : <FontAwesomeIcon icon={faEmptyStar} />
+      )}
+      { abuseFlagged && <FontAwesomeIcon className="my-1 mx-auto" icon={faFlag} /> }
+      <FontAwesomeIcon className="my-1 mx-auto" icon={faEllipsisV} />
```

Comment:
These elements should be buttons.
```suggestion
      {/* Only show the star if the comment has a following attribute, indicating it can be followed */}
      { following !== undefined && (
        following
          ? (
            <Button variant="link" className="p-0" size="xs">
              <FontAwesomeIcon icon={faSolidStar} />
            </Button>
          ) : (
            <Button variant="link" className="p-0" size="xs">
              <FontAwesomeIcon icon={faEmptyStar} />
            </Button>
          )
      )}
      { abuseFlagged && (
        <Button variant="link" className="p-0" size="xs">
          <FontAwesomeIcon icon={faFlag} />
        </Button>
      ) }
      <Button variant="link" className="p-0" size="xs">
        <FontAwesomeIcon icon={faEllipsisV} />
      </Button>
```